libs/mep/ace1052/brick/brick.js

```diff
@@ -79,7 +79,11 @@ function decorateBrickIconStack(el) {
   decorateIconStack(el);
   const icnStk = el.querySelector('.icon-stack-area');
   if (!icnStk) return;
-  icnStk.classList.add('body-xs');
+  // #mweb specific change
+  const bodyClass = Array.from(el.classList).find((cls) => cls.includes('-body')) || 'body-xs';
+  const reversedClass = bodyClass.includes('-body') ? bodyClass.split('-').reverse().join('-') : bodyClass;
+  icnStk.classList.remove(...Array.from(icnStk.classList).filter((cls) => cls.startsWith('body-')));
+  icnStk.classList.add(reversedClass);
```

Comment:
I personally think this is overkill for the purposes of the test, this should suffice:
```suggestion
  icnStk.classList.add('body-s');
```

Reply:
done